Packages/CommonUI/CommonUI/Classes/View/TextField/SecureTextField.swift

```diff
@@ -51,11 +67,16 @@ public struct SecureTextField: View {
 					Spacer()
 						.frame(width: Constants.spacing)
 				}
-				SecureField(placeHolder, text: $secureText)
+				CustomSecureTextField(placeHolder: placeHolder,
+									  text: $secureText,
+									  isRevealed: $isRevealed,
+									  isFocused: onEditingChanged,
+									  onSubmit: onSubmit, textChange: self.textChange)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/e5e0a713c2a0f396e41d6374bb8f07759fda2936/Packages/CommonUI/CommonUI/Classes/View/TextField/SecureTextField.swift#L74"></span>Arguments should be either on the same line, or one per line.<br />
<code>multiline_arguments</code> <code>SecureTextField.swift:74</code></td>
    </tr>
  </tbody>
</table>